web/client/plugins/Maps.jsx

```diff
@@ -60,11 +60,16 @@ const PaginationToolbar = connect((state) => {
     }
     let {start, limit, results, loading, totalCount, searchText} = state.maps;
     let page = 0;
-    //    let total = totalCount || 0;
-    if (results && totalCount) { // must be !==0 and exist to do the division
-        page = Math.ceil(start / limit);
+    let total = totalCount || 0;
```

Comment:
const total = Math.min(totalCount || 0, limit || 0);
const page = (results && total && Math.ceil(start / total)) || 0;

Isn't this simpler?

Let's use const as much as possible. Let is usually not necessary.